### ISSUE TITLE
Output images gets increased contrast

### ISSUE THREAD

**Issue by bobmoff** [devMYC/heif-to-jpeg-demo#1]
I tried your demo and it works fine on your image, but when I tried it on my own the output files gets an increase in contrast ?

Here are the files I tested with:
[heic_files.zip](https://github.com/devMYC/heif-to-jpeg-demo/files/2936357/heic_files.zip)

You have any idea why this is?

**Comment by devMYC** [comment]
![output](https://user-images.githubusercontent.com/6139711/56120494-c79ef400-5fa0-11e9-84b6-560bc6f54d1c.jpg)

@bobmoff Yep, I've tested with your images and the .jpeg version is a bit pale. And I've also tried another online converter which gave me the same result. I have no idea what is the cause of this problem. The example picture comes with this repo is taken from [here](https://github.com/nokiatech/heif/tree/gh-pages/content/images).
